requirements.txt

```diff
@@ -1,3 +1,3 @@
 aiohttp~=3.6.2
 aiofiles-ext~=0.4.0
-v3io~=0.5.0
+v3io~=0.5.3
```

Comment:
I don't think this does anything. ~= already means it takes the latest bugfix version.

Reply:
well i'd put that cause the time fixes i did released in that version.. so don't want anything less than that